### ISSUE TITLE
Bump versions to a 4.6.1 base (Stable) - master branch

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2615]
We have now had 4.6.0-0 (RC7) in testing channel for around 6 weeks:
https://forum.rockstor.com/t/v4-5-testing-channel-changelog/8546/13
with it's promotion to the stable channel as our
"First Stable Poetry build": https://github.com/rockstor/rockstor-core/milestone/19
delayed by around 3 weeks due to the following: 
https://forum.rockstor.com/t/appman-rockstor-com-maintenance-open-collective-integration/8867
which in-turn necessitated some stable activation instructions changes.

Regardless, given 4.6.0-0 (RC7) was promoted to master branch (Stable), we have already begun work on maintenance of this stable branch via the following GitHub milestone:

"4.6.0-0 Maintenance issues": https://github.com/rockstor/rockstor-core/milestone/21

Where our Stable activation instructions update and a subsequent Web-UI SSL cert config and associated test have already been addressed.

As such it is proposed that we now update our versioning ready for the previously planned Stable channel release (with a short stay in testing channel updates rpm repository as per our usual policy).

**Comment by phillxnet** [comment]
This issue is to be closed by version changes (in master branch pyproject.toml) indicating our new stable phase.

**Comment by phillxnet** [comment]
Currently master branch has the following test status following a build.sh procedure:
https://rockstor.com/docs/contribute/contribute.html#code-build
https://rockstor.com/docs/contribute/contribute.html#code-test
```
buildvm:/opt/rockstor # poetry env info

Virtualenv
Python:         2.7.18
Implementation: CPython
Path:           /opt/rockstor/.venv
Valid:          True

cd /opt/rockstor/src/rockstor
buildvm:~ # export DJANGO_SETTINGS_MODULE=settings
buildvm:/opt/rockstor/src/rockstor # poetry run django-admin test -v 2
...
Ran 253 tests in 28.179s

OK
```
We also have a successful rpmbuild and consequent install on an x86_64 Leap 15.4 host following the instructions in our https://github.com/rockstor/rockstor-rpmbuild README.md (given some typos detailed in https://github.com/rockstor/rockstor-rpmbuild/issues/23). With the caveat that the build is incorrectly versioned as we build from GitHub releases that are pending this issues resolution and our consequent rockstor-core GitHub release and versioning workflow that is to follow - that, in-turn, does the related release in our jslibs repo and proposed changes in our rpmbuild repo (which includes the rpm changelog taken from our rockstor-core GitHub release.

**Comment by phillxnet** [comment]
Closing as
Fixed by #2617
